### PR TITLE
add my module gomez

### PR DESCRIPTION
**Pull request by gomez-ashleyhermione**
I added my module and call it to main.py with my basic info and goals.

### CODE REVIEW

File: package/gomez.py [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+            print("My goal is to become a game developer and contribute to open-source projects.")
+        elif choice == '3':
+            os.system('cls')
+            print ("------ Comment from De Leon ------")
+        elif choice == '4':
+                os.system('cls')
+                print ("------ Comment from Esparagoza ------")   
+        elif choice == '5':
+            os.system('cls')
+            print ("------ Comment from Alejandro------")
```
Comment (alejandro-aleckmcklaiyre) [inline package/gomez.py:35]: I will add my own comment here.

**Review by alejandro-aleckmcklaiyre — CHANGES_REQUESTED** [review]

File: package/gomez.py [hunk 2]
```diff
@@ -0,0 +1,34 @@
...
+            print("I am Diploma information technology major, and I enjoy programming and playing games.")
+        elif choice == "2":
+            os.system('cls')
+            print("My goal is to become a game developer and contribute to open-source projects.")
+        elif choice == '3':
+            os.system('cls')
+            print ("------ Comment from De Leon ------")
+        elif choice == '4':
+                os.system('cls')
+                print ("------ Comment from Esparagoza ------")
```
Comment (esparagoza-mikkakette) [inline package/gomez.py]: I will add my comment here.

**Review by esparagoza-mikkakette — CHANGES_REQUESTED** [review]

File: package/gomez.py [hunk 3]
```diff
@@ -0,0 +1,35 @@
...
+                os.system('cls')
+                print ("------ Comment from Esparagoza ------") 
+                print ('Goodluck to your goal')  
+        elif choice == '5':
+            os.system('cls')
+            print ("------ Comment from Alejandro------")
+            print ("Hi! That's nice to know. ")
+        elif choice == '6':
+            os.system('cls')
+            print ("------ Comment from Mosenos ------")
```
Comment (ghost) [inline package/gomez.py:43]: I will add my comment here.

**Review by ghost — CHANGES_REQUESTED** [review]

File: package/gomez.py [hunk 4]
```diff
@@ -0,0 +1,36 @@
...
+        
+        if choice == "1":
+            os.system('cls')
+            print("I am Diploma information technology major, and I enjoy programming and playing games.")
+        elif choice == "2":
+            os.system('cls')
+            print("My goal is to become a game developer and contribute to open-source projects.")
+        elif choice == '3':
+            os.system('cls')
+            print ("------ Comment from De Leon ------")
```
Comment (deleon-jasminerobelle) [inline package/gomez.py:25]: I will add my comment here.

**Review by deleon-jasminerobelle — CHANGES_REQUESTED** [review]

**Review by ghost — APPROVED** [review]

**Review by alejandro-aleckmcklaiyre — APPROVED** [review]

**Review by esparagoza-mikkakette — APPROVED** [review]

**Review by deleon-jasminerobelle — APPROVED** [review]

**Pull request merged**